tests/test_encoder.py

```diff
@@ -66,7 +66,7 @@ def test_encoder(self):
     def test_with_wkt(self):
         self.assertRoundTrip(
             input_geometry="LINESTRING(-71.160281 42.258729,-71.160837 42.259113,-71.161144 42.25932)",  # noqa
-            expected_geometry=[[-71, 43], [-71, 43], [-71, 43]])
+            expected_geometry=[[-71, 42], [-71, 42], [-71, 42]])
```

Comment:
Yea, I must have just taken the output and not verified the geometry values when I put in those assertions in the first place. And then I just copy pasted it to proliferate the issue.